ResetPasswordBundle/src/ResetPasswordTimeHelper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace SymfonyCasts\Bundle\ResetPassword;
+
+class ResetPasswordTimeHelper
+{
+    /**
+     * Convert seconds into a human readable string
+     *
+     * Providing 8100 will return 2 hours 15 minutes
+     */
+    public static function getFormattedSeconds(int $seconds): string
+    {
+
+        $hours = (int) floor($seconds / 3600);
+        $minutes = (int) floor(($seconds / 60) % 60);
+
+        $time = '';
+
+        if ($hours === 1) {
+            $time .= "$hours hour";
+        }
+
+        if ($hours >= 2) {
+            $time .= "$hours hours";
+        }
+
+        if ($minutes > 0) {
+            $time = (self::addSpace($time)) . "$minutes minutes";
+        }
+
+        return $time;
+    }
```

Comment:
The problem with this class is that it doesn't allow for translation. I totally see why you're doing this... and it's nice... but I think we might need to be dumber (and just, in the generated controller code, convert to hours and print in the template). But I'm open to ideas!